openmc/mgxs/library.py

```diff
@@ -859,65 +866,80 @@ def get_xsdata(self, domain, xsdata_name, nuclide='total', xs_type='macro',
             xsdata.zaid = self._nuclides[nuclide][0]
             xsdata.awr = self._nuclides[nuclide][1]
 
+        if subdomain is None:
```

Comment:
Ok, so I wasnt sure about this.  But isn't there something where if you change the value of a parameter from default it will be that way the next time the code is executed, even if the default isnt left behind?  I feel like that used to be a thing